docs/source/tools/sandbox.rst

```diff
@@ -431,18 +426,12 @@ that underlies the index.
 
 These metrics include:
 
-- ``daml.index.db.connection.sandbox.pool.Wait`` (timer): time spent waiting to acquire a connection
-- ``daml.index.db.connection.sandbox.pool.Usage`` (histogram): time spent using each acquired connection
-- ``daml.index.db.connection.sandbox.pool.TotalConnections`` (gauge): number or total connections
-- ``daml.index.db.connection.sandbox.pool.IdleConnections`` (gauge): number of idle connections
-- ``daml.index.db.connection.sandbox.pool.ActiveConnections`` (gauge): number of active connections
-- ``daml.index.db.connection.sandbox.pool.PendingConnections`` (gauge): number of threads waiting for a connection
-
-``daml.index.db.deduplicate_command``
--------------------------------------
-
-A timer. Time spent persisting deduplication information to ensure the
-continued working of the deduplication mechanism across restarts.
+- ``daml.index.db.connection.api.server.pool.Wait`` (timer): time spent waiting to acquire a connection
+- ``daml.index.db.connection.api.server.pool.Usage`` (histogram): time spent using each acquired connection
+- ``daml.index.db.connection.api.server.pool.TotalConnections`` (gauge): number or total connections
+- ``daml.index.db.connection.api.server.pool.IdleConnections`` (gauge): number of idle connections
+- ``daml.index.db.connection.api.server.pool.ActiveConnections`` (gauge): number of active connections
+- ``daml.index.db.connection.api.server.pool.PendingConnections`` (gauge): number of threads waiting for a connection
```

Comment:
I see most of the metric support underscore_delimited_metric_name, but here these are with CamelCase

Reply:
yes, we need to change it at some point

Reply:
I don't think you can change that without making a PR to HikariCP.

Reply:
You are right, this is a lost cause